src/x11_platform.h

```diff
@@ -299,4 +299,6 @@ void _glfwInputErrorX11(int error, const char* message);
 
 void _glfwPushSelectionToManagerX11(void);
 
+XRRScreenResources* _glfwXRRGetScreenResources();
```

Comment:
In C this is __not__ a shorthand for `void` as it is in C++. It's [an older form of function declaration](http://en.cppreference.com/w/c/language/function_declaration) that means the parameter number and types are unknown.